pkg/kopia/command/blob_list.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2022 The Kanister Authors.
```

Comment:
Because most of these commands have just one or two functions at most, we should group them based on their parent commands. For example, the `blob list` and `blob stats` or `maintenance info` and `maintenance set`, etc.